extension/content/tests/factories/index.ts

```diff
@@ -97,7 +97,7 @@ export default class Factory<T, O extends object = object>
         if (has("subfactory", fieldBuilder)) {
           let subfactoryOptions = {};
           if (fieldBuilder.passOptions) {
-            subfactoryOptions = fieldBuilder.passOptions(options, partial);
+            subfactoryOptions = fieldBuilder.passOptions(options, partial, rv);
```

Comment:
:man_shrugging: :)

Reply:
This whole factory thing landed pretty recently, so I'm not surprised to find some issues with it. These changes were needed to make the recipe and the contained revision consistent, which wasn't important before but was for these tests.